### PR TITLE
🔖 Add release ci

### PR DESCRIPTION
**Pull request by boredland**
... and do some release prep.

### CODE REVIEW

File: src/alchemyIds.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  "421613": "arb-goerli",
+  "421614": "arb-sepolia",
+  "534351": "scroll-sepolia",
+  "534352": "scroll-mainnet",
+  "7777777": "zora-mainnet",
+  "11155111": "eth-sepolia",
+  "11155420": "opt-sepolia",
+  "168587773": "blast-sepolia",
+  "999999999": "zora-sepolia",
+  "undefined": "solana-devnet"
```
Comment (sakulstra) [inline src/alchemyIds.ts:61]: doesn't this cause problems down the line? perhaps just remove solana-devnet after fetching (is not an evm and not relevant for anything)

Comment (boredland) [reply src/alchemyIds.ts:61]: I mean: one would have to very purposefully pass the word "undefined" here, so I'd say it's an upstream topic. Until we discover an actual issue, I wouldn't do any filtering here.

**Review by sakulstra — APPROVED** [review]
looks fine to me, i guess having the dist here is a must for the action?

Feels a bit weird, but i guess doesn#t hurt either.

**Comment by boredland** [comment]
> looks fine to me, i guess having the dist here is a must for the action?
> 
> Feels a bit weird, but i guess doesn#t hurt either.

Exactly, actions needs a runnable, dependency free js. I never found a way to do that without checking in or writing plain js, so this is the middle ground.

**Pull request merged**